src/sections/Home/Banner/index.js

```diff
@@ -1,24 +1,25 @@
 import React, { useEffect, useState } from "react";
 import BannerDefault from "../Banner-default";
-// import BannerAlt from "../Banner-alt"; 
+// import BannerAlt from "../Banner-alt";
 
 const Banners = [
   <BannerDefault key={0} />,
-  // <BannerAlt key={1}/> 
+  // <BannerAlt key={1} /> 
 ];
 
 const Banner = () => {
-  const [Banner, setBanner] = useState(0);
+  let initialValue;
+  try {
+    initialValue = sessionStorage.getItem("banner") || 0;
+  } catch (e) {
+    initialValue = 0;
+  }
+  const [Banner, setBanner] = useState(initialValue);
```

Comment:
Also, to have less confusion can we use a different name for the variable? Since the function name is also `Banner`?

Reply:
Any suggestion?

Reply:
I think this can be left as it is, if we change other names.